server/keyspace/tso_keyspace_group.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2023 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package keyspace
+
+import (
+	"context"
+
+	"github.com/pingcap/log"
+	"github.com/tikv/pd/pkg/mcs/utils"
+	"github.com/tikv/pd/pkg/storage/endpoint"
+	"github.com/tikv/pd/pkg/storage/kv"
+	"go.uber.org/zap"
+)
+
+// GroupManager is the manager of keyspace group related data.
+type GroupManager struct {
+	ctx context.Context
+	// store is the storage for keyspace group related information.
+	store endpoint.KeyspaceGroupStorage
+}
+
+// NewKeyspaceGroupManager creates a Manager of keyspace group related data.
+func NewKeyspaceGroupManager(ctx context.Context, store endpoint.KeyspaceGroupStorage) *GroupManager {
+	return &GroupManager{
+		ctx:   ctx,
+		store: store,
+	}
+}
+
+// Bootstrap saves default keyspace group info.
+func (m *GroupManager) Bootstrap() error {
+	defaultKeyspaceGroup := &endpoint.KeyspaceGroup{
+		ID: utils.DefaultKeySpaceGroupID,
+		// TODO: define a user kind type
+		UserKind: "default",
```

Comment:
The doc "Keyspace group membership management APIs" defines the enum values of UserKind. Shall we double-check with serverless service team to say if it's sufficient?

Reply:
Yes. The user kind in this PR is just a placeholder.